spec/integration/outputs/retry_spec.rb

```diff
@@ -30,8 +30,7 @@ def mock_actions_with_response(*resp)
       "manage_template" => true,
       "index" => "logstash-2014.11.17",
       "template_overwrite" => true,
-      "hosts" => get_host(),
-      "port" => get_port(),
+      "hosts" => get_host_port(),
```

Comment:
why do some calls have `()` and some not?  Should this be made more consistent (and I can see that you're following the per-file patterns).